cocos/dragon-bones/ArmatureCache.ts

```diff
@@ -108,14 +111,22 @@ export class AnimationCache {
 
     constructor () {
     }
-
+    /**
+     * @en Initialization.
+     * @zh 初始化。
+     * @param {ArmatureInfo} armatureInfo @en Armature info @zh 龙骨信息
+     * @param {string} animationName @en animation Name @zh 动画名称
```

Comment:
- I am not sure if `{ArmatureInfo}` and `{string}` are needed
- Initialization -> Initialization function.
- animation Name -> Animation name. 
- should add `.` or `。` at the end of a sentence